docs/standard/io/handling-io-errors.md

```diff
@@ -0,0 +1,69 @@
+---
+title: "Handling I/O errors in .NET"
+ms.date: "08/27/2018"
+ms.technology: dotnet-standard
+ms.topic: "article"
+helpviewer_keywords: 
+  - "I/O, exception handling"
+  - "I/O, errors"
+author: "rpetrusha"
+ms.author: "ronpet"
+ms.workload: 
+  - "dotnet"
+  - "dotnetcore"
+---
+# Handling I/O errors in .NET
+
+.NET file system methods can throw the following exceptions:
```

Comment:
I recommend a few weasel words here. The file system methods may throw other exceptions if the environment is under stress (such as OutOfMemoryException) or programmer errors (NullReferenceException). How about:


> .NET file system methods can throw the following exceptions to report I/O errors: